tools/build/pack.py

```diff
@@ -608,8 +608,12 @@ def packCordova(build_json=None, app_src=None, app_dest=None, app_name=None):
 
```

Comment:
Currently Cordova 3.6 does not support arch in pack.py, please provide hint if people use -a when pack cordova 3.6 package.